### DISCUSSION TITLE
What is the last version that is able to run on Windows XP?

### DISCUSSION THREAD

**Discussion by ljenster** [IndySockets/Indy#597] [Q&A]
We are trying to update the indy library but the software is being run on windows XP, I'm curious what the last version of Indy was that was reported to be working on this OS.

**Comment by rlebeau** [comment]
Are you having a specific problem that is preventing you from updating?  AFAIK, the latest version of Indy 10 should work fine on XP, though that hasn't been verified in a long time.  I don't recall off-hand if Indy uses any system APIs that don't exist on XP, but if it does, they should be getting loaded dynamically so Indy can fallback if they are not available.

**Comment by addsuminc** [comment]
Confirming that we've seen no issues with the latest Indy running on XP Pro.

**Comment by ljenster** [comment]
Thanks that's all I needed to hear :)
